layer/nrlf/core/constants.py

```diff
@@ -66,6 +68,8 @@ class Categories(Enum):
     PointerTypes.EOL_CARE_PLAN.value: Categories.CARE_PLAN.value,
     PointerTypes.LLOYD_GEORGE_FOLDER.value: Categories.CARE_PLAN.value,
     PointerTypes.NEWS2_CHART.value: Categories.OBSERVATIONS.value,
+    PointerTypes.ADVANCED_CARE_PLAN.value: Categories.CARE_PLAN.value,
+    PointerTypes.TREATMENT_ESCALATION_PLAN.value: Categories.CARE_PLAN.value,
```

Comment:
It would be good to have a test that fails when a new pointer type gets added without the mapping to the category.

Reply:
For reference this is where it would fail: https://github.com/NHSDigital/NRLF/blob/6adbb5a9f10b7564720b3bf17497f1c3081dc185/layer/nrlf/core/dynamodb/repository.py#L20

Reply:
we dont currently have any tests for the repository file (not sure why tbh) I could add this test as a start ?

Reply:
added some tests